WordPress/src/test/java/org/wordpress/android/ui/activitylog/list/filter/ActivityLogTypeFilterViewModelTest.kt

```diff
@@ -1,22 +1,51 @@
 package org.wordpress.android.ui.activitylog.list.filter
```

Comment:
Question (❓ ) How would the rest of this view model's functionality be tested?

Reply:
I didn't want to test them in this PR since some of the logic can't be easily mocked yet. Bunch of tests were added in https://github.com/wordpress-mobile/WordPress-Android/pull/13440 and more are coming in the following PRs.

Reply:
Yes, yes, thank you! I saw those test in the next PR. ❤️